content/200-orm/100-prisma-schema/20-data-model/65-externally-managed-tables.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: External tables
+metaTitle: Externally managed tables
+metaDescription: How to declare and use externally managed tables in Prisma ORM
+sidebar_class_name: preview-badge
+---
+
+## Overview
+
+
+_Externally managed tables_ (or _external tables_ for short) in Prisma ORM are tables that can be **queried via Prisma Client** but are **ignored by Prisma Migrate**.
+
+Sometimes, you might not want Prisma ORM to manage specific tables—such as ones handled by another team or service.
+
+Some concrete use cases for this are:
+- auth services like Clerk or Auth0 that manage specific tables with user and session data
+- storage services like Supabase Storage with tables for storing metadata about buckets and objects
+- a microservice-based organization where specific teams own specific tables in the database
+
+There may be many other scenarios based on custom organizational constraints or preferences where you may not want Prisma ORM to manage specific tables.
+
+:::note
+Externally managed tables are frequently used in combination with [multi-schema](/orm/prisma-schema/data-model/multi-schema) database setups. However, this is not a hard requirement. You can have only a single schema in your database and also declare externally managed tables within it.
+:::
+
+:::warning
+Externally managed tables are currently in [Preview](/orm/more/releases#preview).
+:::
+
+## Workflow
+
+If you want to use external tables, here's the main workflow:
+1. Declare the name of the external tables in your Prisma Config file
+1. Update your Prisma schema (e.g. via `npx prisma db pull`)
+1. Re-generate Prisma Client with `npx prisma generate`
+1. You can now query the external table using Prisma Client but it will be ignored by Prisma Migrate
+1. When the table gets changed (by whoever owns it):
+    1. Re-introspect your database using `npx prisma db pull` or manually update the models in your prisma file
+    1. Re-generate Prisma Client with `npx prisma generate`
+
```

Comment:
Just noticed that we totally forgot to put anything here in regards to the `setupExternalTables` script and why/when this is needed. 😬